.changeset/brown-cameras-allow.md

```diff
@@ -0,0 +1,8 @@
+---
+'@ag.ds-next/react': patch
+'@ag.ds-next/docs': patch
+---
+
+search-box: Add clear button to `SearchBoxInput` component. Add `defaultValue` support to `SearchBoxInput`.
+
+search-input: Update docs with links to components and patterns. Add `defaultValue` support to `SearchInput`. Add `palette` prop to `SearchInputClearButton` for light and dark theme support.
```

Comment:
Unsure of the intention for changelogs, but since the clear button isn't direclt consumable, perhaps we don't need this `palette` part? or should I remove it for the public log?